src/CoreWCF.Http/src/CoreWCF/Channels/HttpRequestContext.cs

```diff
@@ -353,13 +354,16 @@ protected override HttpInput GetHttpInput()
             }
             public override HttpOutput GetHttpOutput(Message message)
             {
-                if (HttpTransportSettings.KeepAliveEnabled)
+                if (StringComparer.OrdinalIgnoreCase.Equals(Http11ProtocolString, _aspNetContext.Request.Protocol))
                 {
-                    _aspNetContext.Response.Headers["Connection"] = "keep-alive";
-                }
-                else
-                {
-                    _aspNetContext.Response.Headers["Connection"] = "close";
+                    if (HttpTransportSettings.KeepAliveEnabled)
+                    {
+                        _aspNetContext.Response.Headers["Connection"] = "keep-alive";
```

Comment:
@mconnew Nit: `keep-alive` is the default behavior if the `Connection` header is omitted for HTTP/1.1, so it shouldn't be necessary to set it here.